### PR TITLE
docs:add blog github database cicd 1 sql review github actions

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: How to Setup Database CI/CD with GitHub -  1) Enable SQL Review with GitHub Actions
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:2]:
```suggestion
title: How to Setup Database CI/CD with GitHub, Part 1: Enable SQL Review with GitHub Actions
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+author: Ningjing
+published_at: 2022/9/2 13:00:00
+feature_image: /static/blog/github-database-cicd-1-sql-review-github-actions/howto-github-1.webp
+tags: Education
+featured: true
+description: This first article will guide you to enable SQL Review GitHub actions (developed by Bytebase) when there is a pull request in your repository.
+---
+
+In the last article [<The Database CI/CD Best Practice with GitHub>](https://www.bytebase.com/blog/database-cicd-best-practice-with-github), we raised a question: “can we treat database the same way we treat application code?” and presented a complete database CI/CD workflow with GitHub idea.
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:11]:
```suggestion
In the last article [The Database CI/CD Best Practice with GitHub](/blog/database-cicd-best-practice-with-github), we raised a question: "can we treat database the same way we treat application code?" and presented a complete database CI/CD workflow with GitHub.
```

**Review by milasuperstar — APPROVED** [review]

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+![sql-review-guide](/static/blog/github-database-cicd-1-sql-review-github-actions/sql-review-guide.webp)
+
+
+2. Filter with **MySQL**, now you have all the available SQL review rules. Click **Download as YAML** and you will get a YAML file called `sql-review.yml`. Rename it to `bb.sql-review.dev.yml`.
+
+![sql-review-guide-filter](/static/blog/github-database-cicd-1-sql-review-github-actions/sql-review-guide-filter.webp)
+
+
+2. Move the `bb.sql-review.dev.yml` file under root directory. Theoretically, you can put it anywhere within the project, however, if it’s under the `.github/workflow/` folder, GitHub might mistake it as a configuration file which can leads to extra chaos.
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:48]:
```suggestion
2. Move the `bb.sql-review.dev.yml` file under your repo's root directory. Theoretically, you can put it anywhere within the project (just to make sure it's outside of .github/).
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+![move-template](/static/blog/github-database-cicd-1-sql-review-github-actions/move-template.webp)
+
+
+## Step 3 - Configure sql-review.yml
+
+1. Go to [GitHub marketplace](https://github.com/marketplace/actions/sql-review), and follow the **Usage** section: Create a file `sql-review.yml` under the `.github/workflows` folder, and paste the content template.
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:55]:
```suggestion
1. Go to [SQL Review GitHub Action on GitHub marketplace](https://github.com/marketplace/actions/sql-review), and follow the **Usage** section: Create a file `sql-review.yml` under the `.github/workflows` folder, and paste the content template.
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+![sql-review-detail](/static/blog/github-database-cicd-1-sql-review-github-actions/sql-review-detail.webp)
+
+
+5. Because they’re **warning** level, you can still merge the PR.
+
+![sql-review-complete](/static/blog/github-database-cicd-1-sql-review-github-actions/sql-review-complete.webp)
+
+
+6. However, iIf you want this “fail” to block the merge, you can do either of the following:
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md]:
```suggestion
6. However, if you want this check **fail** to block the merge, you can do either of the following:
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+- Adjust the levels on [https://www.bytebase.com/sql-review-guide](https://www.bytebase.com/sql-review-guide) before downloading the template file.
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:92]:
```suggestion
- Adjust the rule to `Error` level on [/sql-review-guide](https://www.bytebase.com/sql-review-guide) before downloading the template file.
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+![configure-guideline](/static/blog/github-database-cicd-1-sql-review-github-actions/configure-guideline.webp)
+
+
+- Or edit the review rule list directly in your YAML file.
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:97]:
```suggestion
- Or edit the level field on the review rule list directly in your YAML file.
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+![edit-yaml](/static/blog/github-database-cicd-1-sql-review-github-actions/edit-yaml.webp)
+
+
+Congratulations! Now you enabled SQL Review GitHub Actions for your GitHub repo. In the next article of this series, we’ll explore the core part: 2) GitHub.com Database GitOps. Stay tuned!
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:102]:
```suggestion
Congratulations! Now you have enabled SQL Review GitHub Actions for your GitHub repo. In the next series, we’ll explore part 2 GitHub.com Database GitOps. Stay tuned!
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+tags: Education
+featured: true
+description: This first article will guide you to enable SQL Review GitHub actions (developed by Bytebase) when there is a pull request in your repository.
+---
+
+In the last article [<The Database CI/CD Best Practice with GitHub>](https://www.bytebase.com/blog/database-cicd-best-practice-with-github), we raised a question: “can we treat database the same way we treat application code?” and presented a complete database CI/CD workflow with GitHub idea.
+
+To bring the idea into reality, we broke it down into a three-part tutorial series <How to Setup Database CI/CD with GitHub>:
+
+1. Enable SQL Review with GitHub Actions (this one)
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:15]: The series outline should be put at the begining, and should include the first article

Example https://www.bytebase.com/blog/database-as-code

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
+---
+title: "How to Setup Database CI/CD with GitHub, Part 1: Enable SQL Review with GitHub Actions"
+author: Ningjing
+published_at: 2022/9/2 13:00:00
+feature_image: /static/blog/github-database-cicd-1-sql-review-github-actions/howto-github-1.webp
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:5]: The URL should be /blog/github-database-cicd-part-1-sql-review-github-actions

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+
+This is a series of articles about Database CI/CD with GitHub
+
+ - [The Database CI/CD Best Practice with GitHub](/blog/database-cicd-best-practice-with-github)
+  
+  
+- How to Setup Database CI/CD with GitHub Part 1-3
+- Part 1: Enable SQL Review with GitHub Actions (this one)
+- Part 2: GitHub.com Database GitOps
+- Part 3: Put Them Together
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:19]: The indentation doesn't look right 
CleanShot 2022-09-02 at 16 10 11

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+
+1. Enable SQL Review with GitHub Actions (this one)
+2. GitHub.com Database GitOps
+3. Put Them Together
+
+You can choose to implement GitHub actions, GitOps, or both by following all three.
+
+This first article will guide you to enable SQL Review GitHub actions (developed by Bytebase) when there is a pull request in your repository.
+
+*Bytebase deployment is not needed for this tutorial.
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:36]:
```suggestion
*Bytebase deployment is not needed for this tutorial*.
```

File: content/blog/github-database-cicd-1-sql-review-github-actions.md [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+
+![configure-guideline](/static/blog/github-database-cicd-1-sql-review-github-actions/configure-guideline.webp)
+
+
+- Or edit the level field on the review rule list directly in your YAML file.
+
+![edit-yaml](/static/blog/github-database-cicd-1-sql-review-github-actions/edit-yaml.webp)
+
+
+Congratulations! Now you have enabled SQL Review GitHub Actions for your GitHub repo. In the next series, we’ll explore part 2 GitHub.com Database GitOps. Stay tuned!
```
Comment (tianzhou) [inline content/blog/github-database-cicd-1-sql-review-github-actions.md:115]:
```suggestion
Congratulations! Now you have enabled SQL Review GitHub Actions for your GitHub repo. In the next series, we’ll explore Part 2: GitHub.com Database GitOps. Stay tuned!
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**